apps/vmq_events_sidecar/src/vmq_events_sidecar_plugin.erl

```diff
@@ -312,7 +312,8 @@ on_deliver(
         {MP, ClientId, normalise(UserName), QoS, unword(Topic), Payload, IsRetain, MatchedAcl,
             Persisted},
         ACL
-    ).
+    ),
+    next.
```

Comment:
why are we explicitly returning `next` only here?

Reply:
the on_deliver hook  stops execution after ok in sidecar `vmq_plugin:all_till_ok(on_deliver, HookArgs1);`.
so to make sure that it reaches to metrics plugin I have added `next`.

other hooks are using `vmq_plugin:all` so they do not need  explicit `next`.